src/python-common/ceph/deployment/service_spec.py

```diff
@@ -509,11 +504,12 @@ def __init__(self,
         #: RADOS namespace where NFS client recovery data is stored in the pool.
         self.namespace = namespace
 
-    def validate_add(self):
-        servicespec_validate_add(self)
+    def validate(self):
```

Comment:
This is not needed anymore:

``` python
class Base:
    def validate(self):
        print('Base validate')

class Child(Base):
    def __init__(self):
        super(Child, self).__init__()

Child().validate()

>>> Base validate
```

Reply:
Err actually I think we still need both .. ?

The base validate checkS the service_id and the child validate checks the pool name..

Reply:
oh wow.. I _somehow_ missed the entire part under `super(NFS..` 

my bad :/